### PR TITLE
Add linux install instructions.

### PR DESCRIPTION
**Pull request by lordbagel42**

### CODE REVIEW

File: tools/linux-installation.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    |- /slimevr-ui_0.0.0_amd64.AppImage
+```
+
+# Running the program
+
+Once everything is all set up, all you need to do to run it is execute the AppImage and it should run everything else on its own.
+
+Note: Only tested on Debian and Ubuntu, if you use Arch please ping Bagel#6705 in the SlimeVR Discord server.
+
+Created by Butterscotch!#2066
```
Comment (Louka3000) [inline tools/linux-installation.md:92]:
```suggestion
*Created by Butterscotch!#2066*
```

**Review by Louka3000 — APPROVED** [review]
lgtm, altho it'd be better with pictures :p

**Comment by lordbagel42** [comment]
will add photos later when I have more time

**Comment by lordbagel42** [comment]
also adding instructions on how to update

**Pull request merged**